src/main/java/io/chrislowe/discordle/database/repository/GameMoveRepository.java

```diff
@@ -1,8 +1,15 @@
 package io.chrislowe.discordle.database.repository;
 
 import io.chrislowe.discordle.database.dbo.GameMove;
+import io.chrislowe.discordle.database.dbo.Guild;
+import io.chrislowe.discordle.database.dbo.User;
 import org.springframework.data.repository.CrudRepository;
 
+import java.util.Optional;
+
 public interface GameMoveRepository extends CrudRepository<GameMove, Integer> {
 
+  Optional<GameMove> findTopByGame_GuildAndUserOrderByDatetimeCreatedDesc(
```

Comment:
Is the _ necessary for the spring data parsing?

Reply:
Done